ssas/service/tokenblacklist.go

```diff
@@ -32,12 +32,12 @@ func init() {
 
 // This function should only be called by main
 func StartBlacklist() {
-	TokenBlacklist = NewBlacklist(defaultCacheTimeout, cacheCleanupInterval)
+	TokenBlacklist = NewBlacklist(context.Background(), defaultCacheTimeout, cacheCleanupInterval)
 }
 
 //	NewBlacklist allows for easy Blacklist{} creation and manipulation during testing, and, outside a test suite,
 //	should not be called
```

Comment:
this is outside the scope of this PR but there seems a trend of test fixtures that aren't within test files. Would it we worthwhile to discuss how we can clean these up?

Reply:
yeah I noticed that as well. We should build up a list and decide how we can move them into a test utilities file or something.